### ISSUE TITLE
"Reset the sim to make more observations" could be voiced more often

### ISSUE THREAD

**Issue by zepumph** [phetsims/friction#207]

**Comment by zepumph** [comment]
@Matthew-Moore240 reported that with voicing it is hard to hear this. In a simple test dragging with a mouse we were unabled to hear this more than once after all atoms evaporated.

**Comment by zepumph** [comment]
I think that this is caused by https://github.com/phetsims/friction/issues/204. I will investigate over there.

**Comment by zepumph** [comment]
I was not caused by that! I made a fix and this should work better in master. 


Please test and see with this https://phet-dev.colorado.edu/html/friction/1.6.0-dev.7/phet/friction_en_phet.html?voicingInitiallyEnabled

**Comment by Matthew-Moore240** [comment]
Okay, I played with it a bit and it seems like the reset sim is getting cut off by the jiggling/cooling responses. So maybe we make the hints assertive? I am not sure.

**Comment by zepumph** [comment]
I think that some of that is because of the workaround we currently have in place for max temp alerts over in https://github.com/phetsims/friction/issues/204. Right now it isn't its own utterance, but I will work on a better workaround now, and see if that helps.

**Comment by zepumph** [comment]
The workaround improvement was helpful, but there was a also a bug to fix in alerting max temp. Now if you stay at max temp you will get a new alert every half second (the time delay and increasing alerts). I think this issue should be fixed now when 

> I played with it a bit and it seems like the reset sim is getting cut off by the jiggling/cooling responses.

The cooling alerts should not be interrupting anything except themselves (like if they repeat).

Please review and see if it sounds better. A really good use case is the just keep dragging quickly to make max friction, and note the repetition of "reset the sim" every third Utterance.

**Comment by Matthew-Moore240** [comment]
Yeah, this seems much better. I think it will work well for now.
